community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -574,12 +574,14 @@ public void resampleOutdatedIndexes()
         return Stream.of( new BooleanResult( true ) );
     }
 
+    private static final String DEFAULT_KEY = " <[9895b15e-8693-4a21-a58b-4b7b87e09b8e]> ";
```

Comment:
Why create a new default key here, instead of using the default one in `Name`?

Reply:
Using the one in Name was what we had before, and it did not work. The code interprets this as a missing value, not an optional value, with the result that we get the exception we get. It has to differ from that one, but be as unlikely to be used as a property key.